superset/models/dashboard.py

```diff
@@ -250,6 +250,16 @@ def changed_by_name(self) -> str:
             return ""
         return str(self.changed_by)
 
+    @property
+    def created_by_name(self) -> str:
+        if not self.created_by:
+            return ""
+        return str(self.created_by)
+
+    @property
+    def owners_by_name(self) -> list[str]:
+        return [owner.get_full_name() for owner in self.owners]
```

Comment:
This looks like more of frontend code than a new model property. If you check other instances of the Metadata component, this is being handled on the frontend as we may have different representations of the possible values.

Reply:
Done